### PR TITLE
logistic_requisition: add buttons

### PR DESCRIPTION
**Pull request by gurneyalex**
on the form view, 2 new buttons to go to the CE and PO

### CODE REVIEW

**Comment by yvaucher** [comment]
Some PEP8 to fix otherwise LGTM

File: logistic_requisition/model/logistic_requisition.py [hunk 1]
```diff
@@ -333,6 +338,69 @@ def button_view_source_lines(self):
...
+            'type': 'ir.actions.act_window',
+        }
+        return action_dict
+
+    @api.one
+    @api.depends('line_ids.cost_estimate_id')
+    def _count_cost_estimates(self):
+        """ Set lr_count field """
+        self.ce_count = len(self._get_relevant_cost_estimates())
+        print 'ce count', self.ce_count
```
Comment (yvaucher) [inline logistic_requisition/model/logistic_requisition.py]: print to remove

**Comment by yvaucher** [comment]
:+1:

**Comment by jgrandguillaume** [comment]
:+1:  Tested, it works fine ! Thanks

**Pull request merged**